### ISSUE TITLE
Instructions for adding/removing crop and ratio sizes from script

### ISSUE THREAD

**Issue by jorgenjanssonlee** [robhabraken/content-hub-focal-point-cropping#12]
On first install of 3.0 versions, it wasn't clear to me why the solution created 8 different crop/ratio links out-of-the-box and how I could adjust this to what we actually want to use.
I later re-discovered https://www.robhabraken.nl/index.php/4255/focal-point-crop-2-0 and realised those are defined in the script  (obvious in hindsight).
This is just a suggestion to add some steps to the ReadMe file on how to edit these, maybe just a link to the 2.0 page would be enough.

**Comment by jorgenjanssonlee** [comment]
Created pull request for Install/Config ReadMe

**Comment by robhabraken** [comment]
Thanks Jorgen! I totally understand why this wasn't clear at first. The blog posts are included in the readme file, but this functionality isn't explained in the readme file at all and also changed over the course of different versions, making the explanation being spread out over multiple blog posts, which isn't very practical.

I have accepted your pull request, thanks for that! And also just committed an improved version, where I copied this section over beneath to the Scripts section, and added a brief explanation on how this works, so you don't need to read all the blogs first. And also kept in a reference to where those blog posts are mentioned (so I can add more later in the intro of the readme file).

See https://github.com/robhabraken/content-hub-focal-point-cropping#configure-desired-public-link-croppings--ratios for the new section based on your feedback. Thanks a lot for your testing and sharing your findings!
